docs/README.md

```diff
@@ -65,6 +65,22 @@ mtLinkSdk.init(clientId, options);
 | options.cobrandClientId (private)          | string                                      | false    |               | <strong>NOTE: This is an internal attribute. Please do not use it unless instructed by your integration representative.</strong><br /><br />Brand Moneytree apps with client's branding. E.g: logo or theme.                                                                                                                                                                                                                 |
+| options.samlSubjectId   | string    | false    |   | Set subject Id for saml session version.                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+
+
+### setSamlSubjectId
+Only includes the parameter when calling authorize, logout, openService methods
+
+<h6>Usage:</h6>
+
+```javascript
+mtLinkSdk.setSamlSubjectId(samlSubjectId);
+```
+
+| Parameter                                  | Type                                        | Required | Default Value | Description                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| ------------------------------------------ | ------------------------------------------- | -------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| sublSubjectId                                   | string                                      | true     |               | <p>Overrides the saml_subject_id parameter</p>                                                                                                                                                                                                                 |
```

Comment:
```suggestion
| samlSubjectId                                   | string                                      | true     |               | <p>Set the saml_subject_id parameter</p>                                                                                                                                                                                                                 |
```

It can't overwrite if no one pass it via the init option.